### PR TITLE
Add aria label to collapse button.

### PR DESCRIPTION
**Pull request by adlius**

### CODE REVIEW

File: lib/osf-components/addon/components/osf-layout/registries-side-nav/template.hbs [hunk 1]
```diff
@@ -22,6 +22,7 @@
             <ExpandButton
                 data-analytics-name={{if this.isCollapsed 'Expand' 'Collapse'}}
                 local-class='Toggle'
+                aria-label={{if this.isCollapsed 'Expand side nav' 'Collapse side nav'}}
```
Comment (jamescdavis) [inline lib/osf-components/addon/components/osf-layout/registries-side-nav/template.hbs]: i18n?

**Comment by adlius** [comment]
@jamescdavis Back to you

File: app/locales/en/translations.ts [hunk 2]
```diff
@@ -1691,6 +1691,10 @@ export default {
                 placeholder: 'Your selections will appear here',
             },
         },
+        'registries-side-nav': {
+            expandSideNav: 'Expand side nav',
```
Comment (jamescdavis) [inline app/locales/en/translations.ts]:
```suggestion
            expandSideNav: 'Expand registration form navigation',
```

File: app/locales/en/translations.ts [hunk 3]
```diff
@@ -1691,6 +1691,10 @@ export default {
...
+            collapseSideNav: 'Collapse side nav',
```
Comment (jamescdavis) [inline app/locales/en/translations.ts]:
```suggestion
            collapseSideNav: 'Collapse registration form navigation',
```

**Pull request merged**